lint_test/function_lines_of_code_test/analysis_options.yaml

```
@@ -6,6 +6,6 @@ custom_lint:
   rules:
     - function_lines_of_code:
       max_lines: 5
-      excludeNames:
-        - "longFunctionExcluded"
-        - "longMethodExcluded"
+      exclude:
+         - method_name: longFunctionExcluded
```

We want to allow specifying names as plain String lists. These will be unspecific, and work for both class names and method/function names.

It will also make the migration for existing users a bit easier. 
```suggestion
         - longFunctionExcluded
```